libcxx/test/std/numerics/rand/rand.eng/rand.eng.lcong/alg.pass.cpp

```diff
@@ -38,12 +38,12 @@ int main(int, char**)
 
     // m might overflow. The overflow is not OK and result will be in bounds
     // so we should use Schrage's algorithm
-    typedef std::linear_congruential_engine<T, (1ull << 32), 0, (1ull << 63) + 1> E2;
+    typedef std::linear_congruential_engine<T, 0x100000000ull, 0, (1ull << 63) + 1ull> E2;
```

Comment:
Why did you change `(1ull << 32)` to `0x100000000ull`?

Reply:
No *specific* reason. Mostly, I was trying to make this code a little cleaner and consistent. I figured that the `(1ull << 63) - 0x1ffffffffull` I added a few lines down would be longer and less clear if written as `(1ull << 63) - (1ull << 33) + 1ull`, and if I'm writing the 31-bit hex literal there, then I might as well do it here as well. I don't mind changing this back if that's what's preferred, though.

Reply:
To me `1ull << 32` is easier to parse. With `0x100000000ull` I need to count the zeros. (Since the code is used in C++11 we can't write `0x1'0000'0000ull`.) So personally I really like the shifted version better.